examples/apps/heart_disease_eval/client/UsingHeartEvalGUI.md

```diff
@@ -0,0 +1,17 @@
+# Using the Heart Evaluation GUI Client
+
+1. Follow instructions in the "Testing" section of the [build document](../../BUILD.md) up to step 2
+2. In Terminal 2, run `cd $TCF_HOME/tests/clients`
+3. Open a new terminal, Terminal 3.
+4. In Terminal 3 run `xhost +local:`
```

Comment:
I followed the steps you mentioned here. Did docker-compose up and started the tcf container and when i tired to run xhost command in Step3, I am getting following error. xhost:  unable to open display ""